src-ts/lib/page-footer/PageFooter.tsx

```diff
@@ -0,0 +1,49 @@
+
+import { FC } from 'react'
+
+import { SocialIconFacebook, SocialIconInstagram, SocialIconLinkedin, SocialIconTwitter, SocialIconYoutube } from '../svgs'
+
+import styles from './PageFooter.module.scss'
+
+export interface PageFooterProps {
+}
+
+const PageFooter: FC<PageFooterProps> = ({
+}: PageFooterProps) => {
+    return (
+        <div className={styles['footer-wrap']}>
+            <div className={styles['footer-inner']}>
+                <div className={styles.utils}>
+                    <div>
+                        <span>© 2022 Topcoder</span>
+                        <a href='#'>Support</a>
+                        <a href='#'>See a Bug?</a>
+                    </div>
+                    <div>
+                        <a href='#'>Terms</a>
+                        <a href='#'>Privacy Policy</a>
+                    </div>
+                </div>
+                <div className={styles.social}>
+                    <a href='https://www.facebook.com/topcoder' target='_blank'>
```

Comment:
Seeing these URLs directly in the `href` makes me wonder if these should be constantized somewhere else rather than just added directly into the `<a>` tags. What do you guys think?

Reply:
Alternatively, if we built a small component for each of the social links that included the `<a>` tag and the svg icon, that might feel a little more encapsulated. I'll let you guys decide if it needs changing -- just a thought.

Reply:
@testflyjets i agree the external URLs shouldn't be hard-coded, but there's really no good place to put them as constants. I really like your 2nd idea of creating social media link components.

Reply:
@vas3a let's create a /src-ts/lib/social directory and put the components in there.